Makefile

```diff
@@ -102,8 +102,10 @@ dep-validate: dep-restore
 	@echo "+ $@"
 	@rm -Rf .vendor.bak
 	@mv vendor .vendor.bak
+	@mv Godeps/Godeps.json .Godeps.json.bak
 	@rm -Rf Godeps
-	@$(GODEP) save ./...
+	@$(GODEP) save $(PKGS)
 	@test -z "$$(diff -r vendor .vendor.bak 2>&1 | tee /dev/stderr)" || \
 		(echo >&2 "+ borked dependencies! what you have in Godeps/Godeps.json does not match with what you have in vendor" && false)
 	@rm -Rf .vendor.bak
+	@mv .Godeps.json.bak Godeps/Godeps.json
```

Comment:
It does generate a new `Godeps.json` but it is different than what is checked in since it lists the packages instead of `./...`. Didn't quite see any other good way to make sure the `make` didn't cause changes picked up by git  and ensure that contrib dependencies are not included. I don't really like this but the way this make command alters the working tree is already kind of awkward.